app/main.py

```diff
@@ -1,3 +1,13 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: object) -> object:
+    cache = {}
+
+    def wrapper(*args) -> str:
```

Comment:
```suggestion
    def wrapper(*args) -> str:
```
yu should return not a string